### PR TITLE
Replace DrawerItems with DrawerNavigationItems for 4.x

### PR DESCRIPTION
**Pull request by ansonhoyt**
When upgrading from 3.x to 4.x, I updated the packages and imports per https://reactnavigation.org/docs/4.x/upgrading-from-3.x/ and everything worked except the drawer nav. When I used `DrawerItems`, I got an error:
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

My suggested change fixed my little Drawer Navigator. I think maybe this component was renamed in `react-navigation-drawer` 1.4.0, which is the version the migration guide recommends for 4.x.

This might be the wrong change. I saw the 2.4.0 [changelog](https://github.com/react-navigation/react-navigation/blob/4.x/packages/drawer/CHANGELOG.md#240-2020-02-24) mentions DrawerItems is re-exposed. In that case, perhaps the migration guide should be updated to recommend installing this newer version?

I lost motivation tracing the source with repos moving around, but hopefully this PR is somewhat helpful.

# READ ME PLEASE!

### TL;DR: Make sure to add your changes to versioned docs

Thanks for opening a PR!

The docs cover several versions of `react-navigation`, and in some cases there are several files (for version 1, version 2 and etc.) that all describe a single page of the docs (eg. "Getting Started").

Please make sure that the edit you're making in `docs/file-you-edited.md` is also included in the file for the correct version, eg. `/versioned_docs/version-3.x/file-you-edited.md` for version 3. If such file doesn't exist, please create it. :+1:

### CODE REVIEW

**Comment by satya164** [comment]
Thanks for the PR. However both are exported in latest versions of drawer so it shouldn't cause any issues: https://github.com/react-navigation/react-navigation/blob/10c6c3280f07949bce0725531d10eee21d9ad5fb/packages/drawer/src/index.tsx#L18

You likely had an older version or outdated cache.

**Comment by ansonhoyt** [comment]
Yes, I did have an older version.

I was following [Upgrading from 3.x](https://reactnavigation.org/docs/4.x/upgrading-from-3.x/) guide, and used the provided command, `yarn add react-navigation react-navigation-stack@^1.7.3 react-navigation-tabs@^1.2.0 react-navigation-drawer@^1.4.0`, which doesn't install the 2.x.x version needed to work. Ideally these docs should work, and a major version jump beyond the documented one was unexpected, but it is a small issue 🤷 😄 

So, the migration guide is slightly mismatched to the working code. I should've suggested a migration docs PR instead.

Thank you.

**Comment by satya164** [comment]
> I was following Upgrading from 3.x guide, and used the provided command, yarn add react-navigation react-navigation-stack@^1.7.3 react-navigation-tabs@^1.2.0 react-navigation-drawer@^1.4.0, which doesn't install the 2.x.x version needed to work.

If you're installing older versions, then it means that you don't need to change any code. The migration guide for navigators are only for when you upgrade navigators (which is in separate upgrading your navigators section).
